qos_capacity.adoc

```diff
@@ -384,37 +384,36 @@ initiating the operation via the `cc_alloc_ctl` register.
 
 Capacity controllers enumerate the allocatable _capacity blocks_ in the `NCBLKS`
 field of the `cc_capabilities` register. The `cc_block_mask` register is
-programmed with a bit-mask where each bit represents a _capacity block_ for the
-operation. A limit on the _capacity unit_, if configuration of such limits is
-supported (i.e., `cc_capabilities.CUNIT=1`), that can be occupied in the
-allocated _capacity blocks_ may be programmed in the `cc_cunits` register. If
-configuration of a limit  on the _capacity units_ is not supported, then the
-controller allows the use of all _capacity units_ in the allocated _capacity
+programmed with a bit-mask value, where each bit represents a _capacity block_ for the
+operation. If configuring _capacity unit_ limits is supported (for example, 
+`cc_capabilities.CUNIT=1`), then the number of allocated capacity blocks can 
```

Comment:
@kersten1 "then the number of allocated capacity blocks can" -> "then a limit on the _capacity units_ that can be occupied in the allocated _capacity blocks_ can"